### PR TITLE
doc(release note): add links to doc pages

### PR DESCRIPTION
**Pull request by NathalieC**
add link to profiles in studio
add link to SAML

### CODE REVIEW

**Comment by tnguyen1** [comment]
retest this please

File: md/release-notes.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ Simply Bonita.
...
-### Industrialization of Living Applications in Bonita Studio: Profiles
-Our goal is to provide you with as few deployment artifacts as possible when creating applications.
-To achieve this, we need all application resources to be available in the Studio / UI Designer development suite.
-After application descriptors, profiles are now available in Bonita Studio.
-In Teamwork editions, you can edit the default profiles mapping with organization entities.
-In Efficiency and Performance editions, you can also create custom profiles, map them to the organization, and create their Bonita Portal menus if needed.
+### [Industrialization of Living Applications in Bonita Studio: Profiles](profileCreation.md)
+Our goal is to provide you with as few deployment artifacts as possible when creating applications.  
+To achieve this, we need all application resources to be available in the Studio / UI Designer development suite.  
+After application descriptors, profiles are now available in Bonita Studio.
```
Comment (benjaminParisel) [inline md/release-notes.md]: I suggest to put link on word [profiles] and not on section title...what did you tkink about this ?

File: md/release-notes.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ Simply Bonita.
...
+After application descriptors, profiles are now available in Bonita Studio.  
+In Teamwork editions, you can edit the default profiles mapping with organization entities.  
+In Efficiency and Performance editions, you can also create custom profiles, map them to the organization, and create their Bonita Portal menus if needed.  
 
 ### Process display name
 It is now possible to add a display name for a process, so the development team can still use the technical "name" for a Pool in Bonita Studio and provide a readable name for users in Bonita Portal. If not set, the technical name is used.
 
-### SAML logout
+### [SAML logout](single-sign-on-with-saml.md)
 SAML logout has been implemented, so that a user logging out of Bonita is also loggued out of all applications managed by the compagny Single-Sign-On system.
```
Comment (benjaminParisel) [inline md/release-notes.md]: I suggest to put link on word [saml] and not on section title...what did you tkink about this ?

**Review by benjaminParisel — COMMENTED** [review]
Just 2 small  comments for you

File: md/release-notes.md (hunk 1 above)
Comment (NathalieC) [reply md/release-notes.md]: ok for me

File: md/release-notes.md (hunk 2 above)
Comment (NathalieC) [reply md/release-notes.md]: good

**Comment by NathalieC** [comment]
Made the changes. Ready for a new review.

**Comment by benjaminParisel** [comment]
LGTM

**Pull request merged**